ReactNative/iTwinRNStarter/android/app/src/debug/java/com/bentley/itwinrnstarter/ReactNativeFlipper.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
```

Comment:
This copyright and LICENSE notice aren't right. We have a LICENSE.md in the root of the source tree, but that MIT license file lists Bentley as the copyright holder, not Meta, and is LICENSE.md, not LICENSE.

Reply:
Almost all the code in the new files was generated by the react-native CLI. In this case, maybe that file shouldn't be committed (it's in a debug folder), I'll need to investigate.

Reply:
The release file has the same problem. If there is a LICENSE file somewhere, then maybe update the comment to indicate where it is.

Reply:
Flipper is apparently an [Extensible mobile app debugger](Extensible mobile app debugger) created by Facebook.

I think these generated files are needed for React Native debugging but I'm not sure if they're optional. They're certainly being used in the current code.

I don't know what to do about the license notice, I'm open to suggestions.